Example/Gluon/GluonApp.swift

```diff
@@ -170,6 +170,56 @@ struct DatePickerModal: LeafComponent {
   }
 }
 
+struct CALayerModal: LeafComponent {
+  struct Props: Equatable {
+    let isPresented: State<Bool>
+  }
+
+  static func render(props: Props, hooks: Hooks) -> AnyNode {
+    let left = hooks.state(0.5 as Float)
```

Comment:
can you please rename this to a more sensible name, maybe just plain `state`? `left` doesn't seem like a good fit as it doesn't seem to be related to any position properties in any way